models/public/licenses/CC-BY-4.0.txt

```diff
@@ -73,7 +73,7 @@ Section 1 -- Definitions.
      and in which the Licensed Material is translated, altered,
      arranged, transformed, or otherwise modified in a manner requiring
      permission under the Copyright and Similar Rights held by the
-     Licensor. For purposes of this Public License, where the Licensed
```

Comment:
not sure that we should change license text

Reply:
Revert changes